v20.1/backup.md

```diff
@@ -171,96 +155,49 @@ Per our guidance in the [Performance](#performance) section, we recommend starti
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP DATABASE bank \
-TO 'gs://acme-co-backup/db/bank/2017-03-29-nightly' \
-AS OF SYSTEM TIME '-10s' \
-INCREMENTAL FROM 'gs://acme-co-backup/database-bank-2017-03-27-weekly', 'gs://acme-co-backup/database-bank-2017-03-28-nightly' WITH revision_history;
-~~~
-
-The examples above show incremental backups [with revision history](#with-revision-history).
-
-### Create locality-aware backups
-
-You can create locality-aware backups such that each node writes files only to the backup destination that matches the [node locality](configure-replication-zones.html#descriptive-attributes-assigned-to-nodes) configured at [node startup](cockroach-start.html).
-
-A locality-aware backup is specified by a list of URIs, each of which has a `COCKROACH_LOCALITY` URL parameter whose single value is either `default` or a single locality key-value pair such as `region=us-east`. At least one `COCKROACH_LOCALITY` must be the `default`.
-
-Backup file placement is determined by leaseholder placement, as each node is responsible for backing up the ranges for which it is the leaseholder.  Nodes write files to the backup storage location whose locality matches their own node localities, with a preference for more specific values in the locality hierarchy.  If there is no match, the `default` locality is used.
-
-{{site.data.alerts.callout_info}}
-Note that the locality query string parameters must be [URL-encoded](https://en.wikipedia.org/wiki/Percent-encoding) as [shown below](#example-create-a-locality-aware-backup).
-{{site.data.alerts.end}}
-
-#### Example - Create a locality-aware backup
-
-For example, to create a locality-aware backup where nodes with the locality `region=us-west` write backup files to `s3://us-west-bucket`, and all other nodes write to `s3://us-east-bucket` by default, run:
-
-{% include copy-clipboard.html %}
-~~~ sql
-BACKUP DATABASE bank TO ('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
+> BACKUP bank.customers \
+TO 'gs://acme-co-backup/bank-customers-2017-03-27-weekly' \
+AS OF SYSTEM TIME '-10s';
 ~~~
 
-The backup created above can be restored by running:
+To backup multiple tables:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-RESTORE FROM ('s3://us-east-bucket', 's3://us-west-bucket');
+> BACKUP bank.customers, bank.accounts \
+TO 'gs://acme-co-backup/database-bank-2017-03-27-weekly' \
+AS OF SYSTEM TIME '-10s';
 ~~~
 
-#### Example - Create an incremental locality-aware backup
+### Create incremental backups
 
-To make an incremental locality-aware backup from a full locality-aware backup, the syntax is just like for [regular incremental backups](#create-incremental-backups):
+<span class="version-tag">New in v20.1:</span> If you backup to a destination already containing a backup, an incremental backup will be produced in a subdirectory with a date-based name (e.g., `destination/day/time_1`, `destination/day/time_2`):
```

Comment:
This won't work for any sinks where we don't have "list" access. So it will work for s3:// and gs:// and nodelocal but not for http://

Do you think that warrants a warning?

Reply:
I'm wary of getting in the weeds of details like that here -- My guess is writable HTTP sinks are extremely rare (we require PUT/WebDAV which is pretty unusual I think?)